### DISCUSSION TITLE
Yuzu is alive!!!

### DISCUSSION THREAD

**Discussion by ksuuk** [Hazuki0x0/YuzuBrowser#201] [General]
Vau, You are back.

Please keep Yuzu alive, as defeloper of the Berry browser (based Yuzu) is ruined the whole Yuzu -s idea, it's not open source, does not include reader mode, hide url/tab etc bar when scrolling page down and developer does not react nor reply at all...

**Comment by Ejimax** [comment]
Hi there. I'm a developer of Berry Browser, the fork of Yuzu Browser.
I'm happy to hear that Yuzu Browser is back in development!

I know some people wonder why Berry Browser is not open source, so I will answer the questions I have received.


> Why Berry Browser is not open source?

About a year ago, I privately forked Yuzu Browser to improve some features, such as ad blocking. Until now, the reason hasn't changed; I am developing the app for my own use.
At that time, I chose to make the project closed source. Making app open source involves responsibilities such as code quality control and issue management.
I am a lazy person, so I want to avoid that such extra work delaying development.


> Why publishing to Google Play?

Google Play is a very good app publishing service. I need user feedback to improve app.


> Why did you remove some features!?

The reason is, difficulty in maintenance. It is a trade-off. I will do my best, but some function is not worth the cost such as maintaining and performance.
Possibly there is another reason: I don't use it.


---
Berry Browser is not a successor of Yuzu Browser. It is just one option and user can choose whether to use. As a companion who shares the same concept, I hope to be respectful and constructive each other.


@ksuuk
So sorry for not responding. I don't have enough time to reply to all messages.

**Comment by ksuuk** [comment]
@Ejimax 

Seeing how often (mostly daily) You update Berry Browser, You can't be called lazy. Also publishing an app on Google Play gives users a legitimate expectation that You listen them and You respond to them. Therefore, please add this explanation to Google Play as well.

**Comment by Ejimax** [comment]
@ksuuk
I understand your opinion, but apps distributed on Google Play are not obliged to listen to every user's wishes, and in fact most apps don't. I am sorry that not all users are satisfied.

It does not mean that all feature requests are rejected, but I determine what to implement (in that sense, not different from Yuzu). Unfortunately your feature request was declined, but there is a reason: difficulty in maintainance.

**Comment by HemanthJabalpuri** [comment]
@Ejimax
> Making app open source involves responsibilities such as code quality control and issue management.
I am a lazy person, so I want to avoid that such extra work delaying development.

Open sourcing does not mean publishing code to Github or whatever. You can also release source code as just an compressed archive like zip(like [this](https://arachnoid.com/android/SSHelper/#Copyright__source__version_history), [this](https://androidfilehost.com/?fid=11410963190603877523)). This will remove your additional work of giving commits, maintaing issue tracker and publishing releases.

There are only few open-source browsers that are lightweight. Eg:- Fulguris, jQuarks, Lightning, Yuzu, FOSS Browser.

Thanks
